helpers/snakeGameGATrain.py

```diff
@@ -91,16 +99,24 @@ def game_over(self):
 
 			self.game_scores = []
 
+			date_time = datetime.now().strftime("%m/%d/%Y %H:%M:%S")
+
 			#Write data about this generation to ga_data.txt
 			file = open("GAdata.txt", "a+")
 			file.write("Generation " + str(self.num_generations) + "\n")
 			file.write("Best Individual: " + str(best_individual) + "\n")
 			file.write("Best Fitness: " + str(best_fitness) + "\n")
 			file.write("Average Fitness: " + str(average_fitness) + "\n")
 			file.write("Average Game Score: " + str(average_game_score) + "\n")
-			file.write("DateTime: " + datetime.now().strftime("%d/%m/%Y %H:%M:%S") + "\n\n\n")
+			file.write("DateTime: " + date_time + "\n\n\n")
 			file.close()
 
+			#Write the same data to a .csv
```

Comment:
Comments should make sense on their own
```suggestion
			#Write data about this generation to clean_ga_data.csv
```